db/migration/0013_previous_lenny_balinger.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "community_district" ALTER COLUMN "li_ft" SET DATA TYPE geometry(multiPolygon,2263);
```

Comment:
For "The migration drops any existing data in the liFt column because they are multipoints" is this something we'd manually add the SQL for here or is there a way to do it with Drizzle?

Reply:
I'll run this locally tomorrow. If it starts shouting errors, we'll have some answers

Reply:
Yes, it shouts errors- specifically about multipoints not being multipolygons. I'll share a code snippet for that.